### PR TITLE
Docker images are now based on openSUSE Leap 15.0

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Docker images are now based on openSUSE Leap 15.0

## GUI diff

After:

- [x] **DONE**

## Documentation
- No documentation needed: CI change.

- [x] **DONE**

## Test coverage
- No tests: CI change

- [x] **DONE**

## Links

Tracks https://github.com/SUSE/spacewalk/issues/6805

- [x] **DONE**

### CODE REVIEW

**Comment by juliogonzalez** [comment]
gitarro no changelog needed !!!

**Comment by juliogonzalez** [comment]
Right, the build still failing, so I will need to add more stuff.

**Comment by juliogonzalez** [comment]
BTW: susemanager_unittests failing are legit, but not related to this PR. It's related to the `spacewalk-repo-sync` issues that are now detected correctly, as we test with Leap 15.0 and python3 :-)

**Comment by cbosdo** [comment]
Works like a charm locally though

**Comment by mcalmer** [comment]
I wonder why the obs ui say that it expire 2021

**Comment by juliogonzalez** [comment]
Adding @lneves12 for the nodejs container problems.

**Comment by lneves12** [comment]
@juliogonzalez  I will try to take a look at the error later today.

I am adding @mateiw as he was the one implementing this part, in case the error ring a bell to him :)

**Comment by mcalmer** [comment]
My key was already re-newed but I had to re-build the repository. osc wipebinaries helped.

I have build and pushed the `uyuni-push-to-obs` container.

**Comment by juliogonzalez** [comment]
`uyuni-push-to-obs` built and pushed to the registry.

Only thing missing right now is the problem with nodejs.

**Comment by juliogonzalez** [comment]
Ping @lneves12, any news?

**Comment by lneves12** [comment]
It seems susemanager-sdk-nodejs-devel is now installing the packages on /usr/lib64 and not /usr/lib.

Anyway I changed the implementation to fix https://github.com/SUSE/spacewalk/issues/6237

cc: @mateiw Can you review the commit: https://github.com/uyuni-project/uyuni/pull/547/commits/cf9fb9dce7c3808679ce91ef00ea31f42d989f5e ?

**Comment by lneves12** [comment]
@juliogonzalez can you rebuild and publish the new nodejs docker files?

File: web/html/src/scripts/docker-javascript-lint.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ zypper --non-interactive in susemanager-nodejs-sdk-devel
 cd /manager/web/html/src
 
 # link to node_modules from susemanager-nodejs-sdk-devel
-ln -s /usr/lib/node_modules/ node_modules
+ln -s /usr/lib64/node_modules/ node_modules
```
Comment (juliogonzalez) [inline web/html/src/scripts/docker-javascript-lint.sh]: As requested by @lneves12.

Unfortunately this is part of spacewalk-web so @cbosdo, you will need to rebuild it in your branch to test. Can you have a look?

**Comment by lneves12** [comment]
Oh @juliogonzalez to fix  https://github.com/SUSE/spacewalk/issues/6237 I ended up solving the problem in a different way: https://github.com/uyuni-project/uyuni/commit/cf9fb9dce7c3808679ce91ef00ea31f42d989f5e

I though I pushed that commit to your fork, but the commit is now gone :O you either force pushed my changes or I messed up while committing :P 

Can you cherry pick that commit?

**Comment by juliogonzalez** [comment]
@lneves12 I am not sure why, but I could not cherry-pick. However I was able to download the patch and apply it to this PR, so everything should be fine now. Please review.

**Comment by lneves12** [comment]
thanks @juliogonzalez everything looks good. After publishing the new nodejs images, the problem should be fixed :)

**Comment by lneves12** [comment]
@juliogonzalez @cbosdo  javascript_lint seems to be fixed :tada:

**Comment by cbosdo** [comment]
@ACK from my side too

**Pull request merged**